### PR TITLE
Add case where m is even and fix normalization

### PR DESCRIPTION
**Pull request by julienguy**
Add case where m is even and fix normalization

### CODE REVIEW

**Comment by julienguy** [comment]
Noting here that this fixes issue #126 .
Here is the result of a toy simulation with one emission line centered at 4000A, for an output spectrum of wavelength bin=0.8A.
new-code

File: specsim/camera.py [hunk 1]
```diff
@@ -305,23 +305,33 @@ def get_output_resolution_matrix(self):
...
+                else : # odd
+                    ic1_ = m // 2 - (k - i)
+                    # Count the rows contributing to each column.
+                    cols_sum[expanded + ic1_] += 1
+                    # Sum the data across rows for each column.
+                    data_sum[expanded + ic1_] += data_in[packed]
+                    num_sum += 1
             # Combine into a single output row.
-            data = data_sum[output_slice].reshape(n, m).sum(axis=1) / m
+            data = data_sum[output_slice].reshape(n, m).sum(axis=1) / num_sum
```
Comment (p-slash) [inline specsim/camera.py:334]: If/else is redundant. Simply can be `data = data_sum[output_slice].reshape(n, m).mean(axis=1) / 2`

 Maybe I'm missing something. Can we just rewrite this as:
```
data = data_sum[output_slice].reshape(n, m).sum(axis=1)
counts = cols_sum[output_slice].reshape(n, m).sum(axis=1)
data /= counts
```
?

Comment (julienguy) [reply specsim/camera.py:334]: go ahead and fix it in your own branch or this one. both are fine.

**Comment by p-slash** [comment]
This fix is implemented to the original branch via commit f24e6f2dab20c9dedf584e482fba03fb0e1eb73c

**Pull request closed without merging**